Adds Bellatrix date warning

wackerow opened the pull request:
<img width="1108" alt="image" src="https://user-images.githubusercontent.com/54227730/186771890-70970f77-4f93-4e29-9ceb-3ea3d1dbd568.png">

The last update to the Merge Readiness page included the TTD and estimated date for Mainnet merge, but did not include a note about the timing of the required Bellatrix upgrade set to occur for the Beacon Chain at epoch 144896. 

- Adds note about upgrading before Bellatrix
- Adds intl support for date/time formatting so these display appropriately in all locales.

CarlBeek reviewed (approved):
Thanks @wackerow, this is a helpful warning & the numbers check out.

One thing I think we should think about is the president of reminding users of upcoming upgrades via the launchpad. Is this something we want to do and maintain moving forward?

wackerow commented on src/pages/MergeReadiness/index.tsx:
```diff
@@ -148,47 +148,90 @@ export const MergeReadiness = () => {
+          </Text>
+          <Text className="mb20">
+            <FormattedMessage
+              defaultMessage="Mainnet total terminal difficulty (TTD) has been set to {difficulty}. Ethereum is estimated to reach this TTD within a day of {ttdDateTime} (follow {link} for latest estimate)."
+              values={{
```
Right, this was the main reason for adding the note about being ready by Sept 6 in this PR (just below this sentence)

Bordel estimates for when we hit TTD haven't changed much, but perhaps we could adjust this to "within a few days of" to give a little more wiggle room on the TTD, but the main deadline for node operators is the 6th... personally not so worried about the exact crossing of the TTD.

wackerow commented:
New additions:
- Added the date range from 9/10 to 9/20 for consistency with other EF communications
- Added link to 797.io/themerge as a second TTD tracker
- Added date information to "When Merge?" bonus section

<img width="930" alt="image" src="https://user-images.githubusercontent.com/54227730/187541886-600c20cc-77fc-49ab-9d56-94583171131f.png">

<img width="914" alt="image" src="https://user-images.githubusercontent.com/54227730/187541846-3e5a857f-d8ee-4ad9-9e87-78e89317fc32.png">

hwwhww reviewed (approved):
lgtm 👍👍
